ui/component.js

```diff
@@ -2570,12 +2570,13 @@ Component.addClassProperties(
                 throw validation;
             }
 
-            for (var key in parameters) {
+            for (const key in parameters) {
```

Comment:
const makes sense here, I'm just wary of the cost of the const being re-created every loop, vs the declaration of a let key first, before the loop or in a for(let key, ..;...) . I know engines are "supposed" to optimize that, I just like to not rely on that as much as possible!